tests/components/ring/test_camera.py

```diff
@@ -30,15 +35,44 @@
 SMALLEST_VALID_JPEG_BYTES = bytes.fromhex(SMALLEST_VALID_JPEG)
 
 
+@pytest.fixture
+def create_disabled_camera_entities(
+    hass: HomeAssistant,
+    mock_config_entry: ConfigEntry,
+    entity_registry: er.EntityRegistry,
+):
+    """Create the entity so it is not ignored by the deprecation check."""
+    mock_config_entry.add_to_hass(hass)
+
+    def create_entry(
+        device_name,
+        description,
+        device_id,
+    ):
+        unique_id = f"{device_id}-{description}"
+        entity_registry.async_get_or_create(
```

Comment:
Why do you create the entity manually?

Reply:
So that they're not disabled by default.

Reply:
There is a test fixture to enable all disabled by default entities. If you need to enable all use that fixture instead

```python
@pytest.mark.usefixtures("entity_registry_enabled_by_default")
```

Reply:
Ah that's great thanks, now updated